examples/protobuf/Thermostat/Client/Program.cs

```diff
@@ -69,20 +80,14 @@ async Task ChangeSetPointAsync(float value)
     Console.ResetColor();
 }
 
-async Task MonitorAsync()
+async Task MonitorAsync(CancellationToken cancellationToken)
 {
```

Comment:
Don't need to use CancelKeyPress we can directly use the provided cancellation token.